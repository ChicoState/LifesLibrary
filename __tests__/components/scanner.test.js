import React from 'react';
import renderer from 'react-test-renderer';
import Scanner from '../../components/scanner.js';
import Enzyme, { shallow, render, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() })

it('render component scanner', () => {
    const TextInputComponent = renderer.create(<Scanner />).toJSON();
    expect(TextInputComponent).toMatchSnapshot();
});

// it('Testing scanner', async () => {
//   const instanceOf = renderer.create(<Scanner />).getInstance();
//   instanceOf.getBook('0439023521', instanceOf.alertInfo);
//   await expect(instanceOf.state.author).toEqual('Suzanne Collins')
// });
