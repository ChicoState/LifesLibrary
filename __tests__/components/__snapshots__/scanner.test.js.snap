// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render component scanner 1`] = `
<ActivityIndicator
  animating={true}
  color="#999999"
  hidesWhenStopped={true}
  size="small"
/>
`;
